### PR TITLE
Feature/examples

### PR DESCRIPTION
**Pull request by gruifin**
-Added new examples (OTAA, ABP, Get-Device-Info)
-Upgraded library ZIP (ThingsUno.zip)

### CODE REVIEW

File: node/TheThingsUno/src/examples/Get-Device-Info/Get-Device-Info.ino [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  delay(2000);
+}
+
+void loop() 
+{
+    debugSerial.println("Device Information");
+    debugSerial.println();
+    ttu.showStatus();
+    debugSerial.println();
+    debugSerial.println("use the device `EUI` to register the device for OTAA \n");
```
Comment (johanstokking) [inline node/TheThingsUno/src/examples/Get-Device-Info/Get-Device-Info.ino]: "Use the `EUI` to register the device for OTAA"

File: node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+TheThingsUno ttu;
+
+void setup()
+{
+  SerialUSB.begin(115200);
+  loraSerial.begin(57600);
+
+  while (SerialUSB) {
+    ;
+  }
```
Comment (johanstokking) [inline node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino]: Why is this here and not in the other version?

File: node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  ttu.showStatus();
+  SerialUSB.println("Setup for The Things Network complete");
+
+  delay(1000);
+}
+
+void loop() {
+  ttu.sendString("Hello world!");
+
+  delay(5000);
```
Comment (johanstokking) [inline node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino]: Set to 20s because of duty cycle. Also for the other example

People will see that the transmission failed, which is not really nice, and it's bad education

File: node/TheThingsUno/src/examples/hello-world-ABP/hello-world-ABP.ino [hunk 4]
```diff
@@ -20,20 +18,20 @@ void setup()
...
 
-  ttu.init(loraSerial, debugSerial);
+  ttu.init(loraSerial, debugSerial); //Initializing...
+
   ttu.reset();
   ttu.personalize(devAddr, nwkSKey, appSKey);
   ttu.showStatus();
 
-  debugSerial.println("Setup for The Things Network.");
+   debugSerial.println("Setup for The Things Network complete");
```
Comment (johanstokking) [inline node/TheThingsUno/src/examples/hello-world-ABP/hello-world-ABP.ino]: Remove space

File: node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino [hunk 5]
```diff
@@ -0,0 +1,34 @@
+#include "TheThingsUno.h"
+
+// Set your app Credentials
+const byte appEui[8] = {0x70, 0xB3, 0xD5, 0x7E, 0xE0, 0xE0, 0x01, 0x4A};
+const byte appKey[16] = {0x73, 0x6D, 0x24, 0xD2, 0x69, 0xBE, 0xE3, 0xAE, 0x0E, 0xCE, 0xF0, 0xBB, 0x6C, 0xA4, 0xBA, 0xFE};
```
Comment (johanstokking) [inline node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino]: Maybe it's better to not use real EUIs and keys here.

**Comment by johanstokking** [comment]
Looks good, just a few small things

File: node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino (hunk 5 above)
Comment (gruifin) [inline node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino]: so instead an empty byte array: 0x00, 0x00..... ?

Comment (johanstokking) [inline node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino]: No leave it as non-compilable, e.g. {` <insert AppEUI here> }`

Add an example as comment, with blanks indeed, for the format and length

Op 23 mei 2016 om 15:43 heeft gruifin <notifications@github.com<mailto:notifications@github.com>> het volgende geschreven:


In node/TheThingsUno/src/examples/hello-world-OTAA/hello-world-OTAA.ino<https://github.com/TheThingsNetwork/sdk/pull/3#discussion_r64224275>:

> @@ -0,0 +1,34 @@
> +#include "TheThingsUno.h"
> +
> +// Set your app Credentials
> +const byte appEui[8] = {0x70, 0xB3, 0xD5, 0x7E, 0xE0, 0xE0, 0x01, 0x4A};
> +const byte appKey[16] = {0x73, 0x6D, 0x24, 0xD2, 0x69, 0xBE, 0xE3, 0xAE, 0x0E, 0xCE, 0xF0, 0xBB, 0x6C, 0xA4, 0xBA, 0xFE};


so instead an empty byte array: 0x00, 0x00..... ?

-
